packages/slate-plugins/src/paste-md/deserializeMd.ts

```diff
@@ -0,0 +1,74 @@
+import { jsx } from 'slate-hyperscript';
+
+export const ELEMENT_TAGS: any = {
+  A: (el: any) => ({ type: 'link', url: el.getAttribute('href') }),
+  BLOCKQUOTE: () => ({ type: 'quote' }),
+  H1: () => ({ type: 'heading-one' }),
+  H2: () => ({ type: 'heading-two' }),
+  H3: () => ({ type: 'heading-three' }),
+  H4: () => ({ type: 'heading-four' }),
+  H5: () => ({ type: 'heading-five' }),
+  H6: () => ({ type: 'heading-six' }),
+  IMG: (el: any) => ({ type: 'image', url: el.getAttribute('src') }),
+  LI: () => ({ type: 'list-item' }),
+  OL: () => ({ type: 'numbered-list' }),
+  P: () => ({ type: 'paragraph' }),
+  PRE: () => ({ type: 'code' }),
+  UL: () => ({ type: 'bulleted-list' }),
```

Comment:
Can you use the constant types? e.g. `LINK`

Reply:
of course! gonna check the constant types available :)